### PR TITLE
docs: document resetting metadata (close #3874)

### LINKED ISSUE #3874: docs: reload and reset metadata from the console
Currently, docs only explain the API and the CLI.

We should also put screenshots of the console so it's easier to show.

### PR DESCRIPTION
**Pull request by ringfungi**
Added a section about resetting metadata, as per the request in http://github.com/hasura/graphql-engine/issues/3874

### Description
The "Managing Hasura metadata" section of Hasura Core docs was missing instructions on how to reset metadata. I added that section.

### Affected components

- [x] Docs

### Related Issues
#3874

#### Catalog upgrade
Does this PR change Hasura Catalog version?
- [X] No

Does this PR add a new Metadata feature?
- [X] No
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [X] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [X] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [X] Not required

#### GraphQL
- [X] No new GraphQL schema is generated

#### Breaking changes

- [X] No Breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: add `<3 hasura` section to view updates and notifications from Hasura (#5070)
 - docs: add docs page on networking with docker (close #4346) (#4811)
 - docs: add postgres concepts page to docs (close #4440) (#4471)
+- docs: add reset metadata guide to docs (close #3874) (#5961)
```
Comment (marionschleifer) [inline CHANGELOG.md:89]: We don't need a changelog entry for this. I added the `no-changelog-required` label. Please remove this.

File: docs/graphql/core/migrations/manage-metadata.rst [hunk 2]
```diff
@@ -157,6 +157,41 @@ when a new column has been added to a table via an external tool such as ``psql`
    Reloading may result in inconsistent metadata status. You may need to resolve
    all inconsistent objects manually or delete them. After that, you need to reload
    metadata again.
+   
+   
+ .. _reset_metadata_manual:
```
Comment (marionschleifer) [inline docs/graphql/core/migrations/manage-metadata.rst:162]: There is something wrong with the indentation here. It messes up the note in the previous section:

image

File: docs/graphql/core/migrations/manage-metadata.rst [hunk 3]
```diff
@@ -157,6 +157,41 @@ when a new column has been added to a table via an external tool such as ``psql`
...
+
+Reseting Hasura metadata
+-------------------------
+
+Clearing GraphQL Engine's metadata is an irreversible process, so use this functionality carefully.
+
+.. rst-class:: api_tabs
+.. tabs::
+
+  .. tab:: CLI
```
Comment (marionschleifer) [inline docs/graphql/core/migrations/manage-metadata.rst:172]: Please make the tab in the same order as in the other examples.

File: docs/graphql/core/migrations/manage-metadata.rst [hunk 4]
```diff
@@ -157,6 +157,41 @@ when a new column has been added to a table via an external tool such as ``psql`
...
+
+  .. tab:: Console
+
+     1. Click on the settings (⚙) icon at the top right corner of the console screen.
+     2. Click on ``Reset`` button.
+     3. A notification should appear prompting you to write "CONFIRM" to conclude the process.
+
+  .. tab:: API
+
+     .. code-block:: http
```
Comment (marionschleifer) [inline docs/graphql/core/migrations/manage-metadata.rst:184]: Please make the example as a `curl` command, like in the one above.

**Review by marionschleifer — CHANGES_REQUESTED** [review]
@ringfungi thank you for your contribution 🙂  I've made some comments. When those things are fixed, we can get this merged.

File: docs/graphql/core/migrations/manage-metadata.rst (hunk 3 above)
Comment (ringfungi) [reply docs/graphql/core/migrations/manage-metadata.rst:172]: I'm not sure what you meant. The order's CLI, console, API, as in the other examples.

**Review by marionschleifer — APPROVED** [review]

**Review by marionschleifer — APPROVED** [review]

**Review by rikinsk — APPROVED** [review]

**Comment by mehtavishwa30** [comment]
Hello @ringfungi!

Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation. If you could please share your email address, we will send you an email with the swag form shortly. ✨

**Comment by ringfungi** [comment]
> Hello @ringfungi!
> 
> Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation. If you could please share your email address, we will send you an email with the swag form shortly. ✨

@mehtavishwa30 Hey!
My email is ringfungi@gmail.com. Thanks.

**Pull request merged**